src/pages/index.js

```diff
@@ -4,7 +4,37 @@ import Layout from '@theme/Layout';
 import useDocusaurusContext from '@docusaurus/useDocusaurusContext';
 import styles from './styles.module.css';
 import repos from '@site/static/repos.json';
+import TwitterTimeline from './TwitterTimeline';
 
+function Social() {
+    return (
+        <section className={styles.social}>
+            <div className="container">
+                <div className="row">
+                    <div className={clsx('col col--4', styles.blog)}>
+                        <div className="text--center">
+                            <p>Placeholder for blog</p>
+                        </div>
+                    </div>
+                    <div className={clsx('col col--4', styles.careers)}>
+                        <div className="text--center">
+                            <p>Placeholder for careers</p>
+                        </div>
+                    </div>
+                    <div className={clsx('col col--4', styles.twitterTimeline)}>
+                        <div className="text--center">
+                            <TwitterTimeline/>
+                            <div className={styles.twitterText}>
+                                <p>What we're saying on Twitter</p>
+                                <a className={styles.arrowLink} href="https://twitter.com/ExpediaGroupEng">→</a>
```

Comment:
Minor: the url `https://twitter.com/ExpediaGroupEng` is repeated twice, both in `index.js` and `TwitterTimeline.js`. Maybe we could add a `customFields.twitter.url` in `docusaurus.config.js` to avoid duplication? Your `TwitterTimeline` component would then receive it as input.
WDYT ?

Reply:
Good suggestion! I've created a section of custom fields for any social links we'll add in `docusaurus.config.js`.
Let me know if it makes sense.

Reply:
Perfect, thanks!